e2e/test/scenarios/embedding/reproductions/37914-embed-preview-parameter-values-not-working.cy.spec.js

```diff
@@ -52,7 +53,9 @@ const dashboardDetails = {
 
 describe("issue 37914", () => {
   beforeEach(() => {
-    cy.intercept("GET", "/api/embed/dashboard/**").as("getEmbed");
+    cy.intercept("GET", "/api/preview_embed/dashboard/**").as(
```

Comment:
The preview didn't seem to work in Cypress before. I've fixed that in `frontend/src/metabase/lib/dom.js`